src/core/Core/Application.cpp

```diff
@@ -140,6 +140,39 @@ ExitStatus App::Application::run() {
         float lineThickness = 6.0f;
         draw_list->AddLine(ImVec2(canvas_p0.x, origin.y), ImVec2(canvas_p1.x, origin.y), IM_COL32(0, 0, 0, 255), lineThickness);
         draw_list->AddLine(ImVec2(origin.x, canvas_p0.y), ImVec2(origin.x, canvas_p1.y), IM_COL32(0, 0, 0, 255), lineThickness);
```

Comment:
Move these two lines (that draw the axes) to just below the gridline loops, to draw the axes over the gridlines. Currently, the gridlines are drawn over the axes, and it looks a bit ugly